### ISSUE TITLE
May I ask about the configs of pre-training? For example, did you use dropout?

### ISSUE THREAD

**Issue by joytianya** [openlm-research/open_llama#21]
May I ask about the configs of pre-training? For example, did you use dropout?

**Comment by young-geng** [comment]
We use the exact same hyperparameters as described in the original llama paper.

**Comment by joytianya** [comment]
I didn't find dropout in the paper. May I ask if llama has used dropout?
